### ISSUE TITLE
REST APIs: unable to call a function that shares the same @post('/suggestion_sys')

### ISSUE THREAD

**Issue by valin1** [e-mission/e-mission-server#626]
Currently when the phone tries to call getSingleTripSuggestion(tripid), the server redirects the phone to getSuggestion() is there. To fix this problem, is the @post('/suggestion_sys') for the getSingleTripSuggestion(tripid) supposed to point to another path like @post('/suggestion_sys/tripid')? 

I was wondering if there are any helpful REST API articles to solve this problem?

**Comment by shankari** [comment]
Yes, the REST API maps a path to a particular function. If you want a different function, you need a different path. There is no method overloading :)
Also, if you want an argument for a function, it has to either come in through the path explicitly, or you need to read it from the request headers/body.

You might find this 
https://bottlepy.org/docs/dev/routing.html
helpful
